src/TranslatableFieldMixin.php

```diff
@@ -45,16 +46,19 @@ public function translatable()
                 } catch (Exception $e) {
                 }
 
-                $value = array_map(function ($val) {
-                    return !is_numeric($val) ? $val : (float) $val;
-                }, (array) $value);
+                if (!empty($value)) {
+                    $value = array_map(function ($val) {
+                        return !is_numeric($val) ? $val : (float) $val;
+                    }, (array) $value);
+                }
 
+                $request = app(NovaRequest::class);
                 $this->withMeta([
                     'translatable' => [
                         'original_attribute' => $this->attribute,
                         'original_component' => $component,
                         'locales' => $locales,
-                        'value' => $value
+                        'value' => $value ?: $this->resolveDefaultValue($request) ?? "",
```

Comment:
Kas siia saaks täpsustavad sulud, et mis järjekorras asjad jooksevad? Mul vahest mingi PHP versiooni/configa see warninguid visanud.

A la `$value ?: ($this->resolveDefaultValue($request) ?? '')`